photo-points/Controllers/AdminController.cs

```diff
@@ -17,13 +17,13 @@ public class AdminController : Controller
     {
 
 
-        //private PhotoDataContext _pdc;
+        private PhotoDataContext _pdc;
 
-        //public AdminController(PhotoDataContext pdContext)
+        public AdminController(PhotoDataContext pdContext)
 
-        //{
-        //    _pdc = pdContext;
-        //}
+        {
+            _pdc = pdContext;
+        }
```

Comment:
I believe we do need it, I forgot to uncomment it before the last push. Thanks for catching that. I had to comment it out to get my piece of the code to work.

Reply:
Sara's question is a good one. The plan is to inject a reference to IAdminReviewServices here instead of the pdContext. The AdminReviewService will use the repository instead of directly accessing the database.

Reply:
I've gone ahead and changed this to IAdminReviewServices.